tests/test_client.py

```diff
@@ -346,6 +346,52 @@ def test_get_decisions(self):
             assert(response.is_ok())
             assert(response.body['data'][0]['id'] == 'block_user')
 
+    def test_get_decisions_entity_session(self):
+        mock_response = mock.Mock()
+        get_decisions_response_json =  \
+            '{' \
+                '"data": [' \
+                    '{' \
+                        '"id": "block_user",' \
+                        '"name" : "Block user",' \
+                        '"description": "user has a different billing and shipping addresses",' \
+                        '"entity_type": "session",' \
+                        '"abuse_type": "legacy",' \
```

Comment:
Also, we only have session decision associated with `account_takeover` today. Its logic may change in the future, and it isn't really related to the test case, either, so you may leave it as is now. It just doesn't look real.

Reply:
agree that it would be nice to change this to `account_takeover` to reflect a correct integration pattern